python/shotgun_desktop/initialization/install.py

```diff
@@ -190,13 +195,16 @@ def create_structure(self):
             fh.write(self._executables[x])
             fh.close()
 
+    def _get_app_store_proxy(self):
+        return self._app_store_http_proxy or self._sg_proxy
```

Comment:
Thinking through the cases here, do you think there is a situation where you'd want to go through a proxy to get to your shotgun site but not to get to the app store?

I'm not sure that would ever happen (although maybe if you need a proxy to get at a local Shotgun site from a machine that has internet access), but if so feels like we should make it so that
app_store_http_proxy: null
is different than it not being in the file at all...

In one case you'd explicitly set it to nothing in the other, you'd inherit http_proxy.

What do you think?